app/server.go

```diff
@@ -827,6 +838,59 @@ func doSessionCleanup(s *Server) {
 	s.Store.Session().Cleanup(model.GetMillis(), SESSIONS_CLEANUP_BATCH_SIZE)
 }
 
+func doStoreAndCheckNumberOfActiveUsersWarnMetricStatus(s *Server) {
+	// uncomment this if we want to have this check only for TE
+	// license := s.License()
+	// if license != nil {
+	// 	mlog.Debug("License is present, skip this check")
+	// 	return
+	// }
```

Comment:
We should only warn for unlicensed correct?

Reply:
yes, I kept it as a reminder, since we might use the check for other metrics.

Reply:
uncommented